modules/ROOT/pages/_partials/include-manage-proxies.adoc

```diff
@@ -3,10 +3,25 @@
 
 Because the monitoring agent requires a SOCKS5 proxy, you need two proxy types if you use Anypoint Monitoring.
 
+[NOTE]
+Starting Runtime Fabric version 2.6.22, the following HTTP proxy is needed for Anypoint Monitoring sidecar or agents as well. SOCK5 proxy is not required for Mule apps deployed on Runtime Fabrics with version less than 2.6.22.
+
 The SOCKS5 proxy must connect to Anypoint Monitoring over TCP at the following endpoints:
 
-* US control plane: `dias-ingestor-router.us-east-1.prod.cloudhub.io`
-* EU control plane: `dias-ingestor-router.eu-central-1.prod-eu.msap.io`
+* US control plane: +
```

Comment:
@bin-shi-mulesoft Can you plase also eye ball these endpoints once.

Reply:
@hegderohit89 The ingestion /api/v2 endpoints are correct. I notice there is no dias-ingestor-nginx for the prod so I need to check whether it is not needed or missing.

@michael-lee-mulesoft Do you know whether whether it needs a `dias-ingestor-nginx` for prod-us, and what is the endpoint? If we don't need it, can we also remove the one for EU? @parepallykiran

Reply:
Previous RTF versions need it. (dias-ingestor-nginx.prod.cloudhub.io for US) New versions won't need it any more.

Reply:
Thanks. So let's get the legacy endpoint removed from the EU control plane as well.